openbase/bco
# Improve Autostart handling

### :scroll: Description

Changes proposed in this pull request:
* Improve Autostart handling by making sure agents and apps are automatically started once the auto start flag has been set.

### How to Test

* Set autostart of an app or agent that has not already been started.
* Make sure the app / agent has started afterwards.
